torch/_tensor.py

```diff
@@ -539,6 +539,55 @@ def register_hook(self, hook):
         self._backward_hooks[handle.id] = hook
         return handle
 
+    def register_post_accumulate_grad_hook(self, hook):
+        r"""Registers a backward hook that runs after grad accumulation.
+
+        The hook will be called after the .grad field is set on a Tensor, meaning
+        all gradients have already been accumulated. The hook should have the
```

Comment:
The hook will be called after all gradients for a tensor have been accumulated, meaning that the .grad field has been updated on that tensor.